vardefunc.py

```diff
@@ -3,26 +3,25 @@
 """
 import math
 import subprocess
-from typing import Tuple, Callable, Dict, Any, cast, Union
 from functools import partial
+from typing import Any, Callable, Dict, Tuple, Union, cast
 
 import fvsfunc as fvf
 import havsfunc as hvf
-import placebo
-
-from vsutil import depth, get_depth, get_y, get_w, split, iterate
 import vapoursynth as vs
+from vsutil import depth, get_depth, get_w, get_y, iterate, split
```

Comment:
Why not order alphabetically here as well?

Reply:
That *is* alphabetically ordered 🤔

Reply:
ah yes indeed lmao